src/Commands/About/AboutVersion.php

```diff
@@ -50,7 +53,14 @@ protected function initialize(InputInterface $input, OutputInterface $output): v
 
             // get module's information from the Core, not the Adapter, not the legacy, this is the correct way.
             $this->moduleRepository = $this->getContainer()->get('prestashop.core.admin.module.repository');
-            if (!$this->moduleRepository instanceof ModuleRepositoryInterface) {
+
```

Comment:
Is there really a risk that the module repository doesn't get defined ?
If not, there's no real need to check the type and throw an exception. But maybe I'm missing something 🤔 .

Reply:
Honestly I just kept the behavior without thinking more about it. If you want me to remove it, tell me and this test disappear ! ^^

Reply:
Right  I was not sure about it.
And in fact, maybe it's my code 🤣 but it makes no sense today.
Whatever.